spec/models/legislator_spec.rb

```diff
@@ -181,6 +181,24 @@
 
   end
 
+  describe "#image_url" do
+    before do
+      @legislator = FactoryGirl.build(:legislator)
+      @bioguide_id = 'b1234'
+      allow(@legislator).to receive(:bioguide_id).and_return(@bioguide_id)
+    end
+    it "calls bioguide_id" do
+      @legislator.image_url
+      expect(@legislator).to have_received(:bioguide_id)
+    end
+    it "calls a proper url" do
+      aws_url = 'http://some-aws-url.com/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.